cg/store/crud/read.py

```diff
@@ -1031,20 +1051,8 @@ def cases_to_analyse(self, workflow: Workflow = None, limit: int = None) -> list
             filter_functions=case_filter_functions,
```

Comment:
can we add a type hint for the variable `cases` in line 1029?

Reply:
Fixed